.eslintrc

```
@@ -0,0 +1,10 @@
+{
+  "extends": "@medic",
```

```suggestion
  "extends": ["@medic/eslint-config"],
```

Do you know what is the difference between this extension and `@medic/eslint-config` ?

---

I think they are the same thing, but we're using @medic in other places: https://github.com/medic/cht-core/blob/master/.eslintrc#L2

---

Cool, sorry I was going to send all my comments in a single submission but I press the wrong button :man_shrugging: , reviewing now ...